packages/resolve-dist/lib/index.ts

```diff
@@ -22,5 +22,5 @@ export const getPathToDesktopIndex = (pkg: 'desktop-gui' | 'launchpad') => {
   // TODO: check if there's a better approach to fix here
   if (pkg === 'launchpad' && !process.env.CI) return `http://localhost:3001`
 
-  return `file://${path.join(__dirname, '..', '..', pkg, 'dist', 'index.html')}`
+  return `file://${path.join(__dirname, '..', '..', 'launchpad', 'dist', 'index.html')}`
```

Comment:
We should be able to leave this as `pkg`? We could use the `process.env.LAUNCHPAD` env var where we call `getPathToDesktopIndex` to pass the correct value in. This seems like an un-necessary change.

I think we should keep the existing stuff working for a long as possible, only breaking things where it's truly necessary. It's useful to be able to spin up the existing app to verify how things work, etc, without changing branches.

Reply:
Agree, was there a reason we needed to change this

Reply:
I was hoping to fully cut over to the launchpad instead of having to keep two code branches alive and then cleanup all of the useless code at the end.